html_css/intermediate_css/css_math_functions.md

```diff
@@ -103,10 +103,17 @@ You may not find a lot of use for max at first, but it is good tool to be aware
 #### clamp()
 
 `clamp()` is a great way to make elements fluid and responsive.
-`clamp()` takes 3 values.
-`font-size: clamp(320px, 80vw, 60rem);`
+`clamp()` takes 3 values:
 
-1. `the smallest value`, 2. `the ideal value`, 3. `the largest value`
+~~~css
+font-size: clamp(320px, 80vw, 60rem);
+~~~
```

Comment:
Can this CSS property be nested in a CSS selector? `h1`, `h2`, or `h3`, any of these should be good.

Reply:
Yes, I have added a h1 selector to the clamp() CSS code example.